### PR TITLE
#794 New ACTIONS - ENTER DATE OF DEATH script

### PR DESCRIPTION
**Pull request by MDR22**
Created new script ACTIONS - ENTER DATE OF DEATH to replace NOTES - DECEASED CLIENT SUMMARY. New script aligns with POLI/TEMP02.08.008.
- Retired NOTES - DECEASED CLIENT SUMMARY, removed instructions from SharePoint
- Updated CLOS with ACTIONS - ENTER DATE OF DEATH, added instructions to SharePoint

### CODE REVIEW

File: actions/enter-date-of-death.vbs [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+
+'Determine if date of death is prior to arrival date for HH Membs other than 01
+If left(household_member_that_died, 2) <> "01" Then
+  Call navigate_to_MAXIS_screen("STAT", "MEMB")
+  'Navigate to HH Memb that died
+  EMWriteScreen left(household_member_that_died, 2), 20, 76
+  transmit
+  EMReadScreen arrival_date_memb_panel, 8, 4, 73
+  'Convert arrival date to MM/DD/YYYY format
+  arrival_date_memb_panel = Left(replace(arrival_date_memb_panel, " ", "/"), 6) & "20" & right(arrival_date_memb_panel, 2)
```
Comment (MeganGeissler) [inline actions/enter-date-of-death.vbs:188]: If there is no arrival date you receive an error. Not sure if you'd run into that in real life cases...but might need handling.
image

**Review by courtrightd — APPROVED** [review]
Looks good, tested good, approved, but can we get a changelog in the menu to announce the retirement and replacement of the script please?

File: actions/enter-date-of-death.vbs (hunk 1 above)
Comment (MDR22) [reply actions/enter-date-of-death.vbs]: Good catch, I resolved this issue.

**Comment by MDR22** [comment]
> Looks good, tested good, approved, but can we get a changelog in the menu to announce the retirement and replacement of the script please?

Done - thanks!

**Pull request merged**